src/main/groovy/org/pih/warehouse/product/ProductPackageCommand.groovy

```diff
@@ -0,0 +1,38 @@
+package org.pih.warehouse.product
+
+import grails.validation.Validateable
+import org.pih.warehouse.core.UnitOfMeasure
+import util.ConfigHelper
+
+class ProductPackageCommand implements Validateable {
+
+    ProductSupplier productSupplier
+
+    UnitOfMeasure uom
+
+    // Package size
+    Integer productPackageQuantity
+
+    BigDecimal minOrderQuantity
+
+    // Package price
+    BigDecimal productPackagePrice
+
+    BigDecimal contractPricePrice
+
+    Date contractPriceValidUntil
+
+    static constraints = {
+        productPackageQuantity(min: 0)
+        minOrderQuantity(nullable: true, min: 0.0)
+        productPackagePrice(nullable: true, min: 0.0)
+        contractPricePrice(nullable: true, min: 0.0)
+        contractPriceValidUntil(nullable: true, validator: { Date date ->
+            Date minDate = ConfigHelper.getMinimumExpirationDate()
+            if (date < minDate) {
+                return false
+            }
+            return true
```

Comment:
can't you just return `date < minDate`?

Reply:
I probably could but would have to negate it, because otherwise I’d return true if date < minDate